libyul/backends/evm/OptimizedEVMCodeTransform.cpp

```diff
@@ -490,14 +490,7 @@ void OptimizedEVMCodeTransform::operator()(CFG::BasicBlock const& _block)
 			createStackLayout(debugDataOf(_functionReturn), exitStack);
 			m_assembly.appendJump(0, AbstractAssembly::JumpType::OutOfFunction);
 		},
-		[&](CFG::BasicBlock::Terminated const&)
-		{
-			// Assert that the last builtin call was in fact terminating.
```

Comment:
Yeah - if we wanted to, we could of course drag along the entire logic into the code transform as well and keep validating things here - but I guess it's fine to trust that it's ok.

Reply:
I added at least some validation back, even though it relies on ``functionCall->canContinue`` to be correct. But conceptually that's fine - the ``canContinue`` flag is part of the CFG and the code transform only guarantees correctness relative to the CFG...